packages/hdwallet-native/src/binance.ts

```diff
@@ -35,16 +37,57 @@ export function MixinNativeBinanceWalletInfo<TBase extends core.Constructor>(Bas
       // Only support one account for now (like portis).
       return undefined;
     }
+
+    binanceDescribePath(path: BIP32Path): PathDescription {
```

Comment:
Would like to see this in core/binance.ts

Reply:
it's currently only implemented in native and keepkey so I wasn't sure if it was appropriate to have it in core if it doesn't apply to all wallets. What do you think?

Reply:
That was the pattern I moved towards for native and think it makes more sense since it should be the same for all wallets. Also if we add to ledger or trezor, we won't need to copy it there as well. Would be my preference.